subprojects/performance/src/templates/cpp-source/src.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
```

Comment:
We should use header guards here (even though `#pragma once` is a lot nicer) to be consistent with our other samples and to match what real projects would normally do.